Misc/NEWS.d/next/Core and Builtins/2024-04-13-16-55-53.gh-issue-117536.xkVbfv.rst

```diff
@@ -0,0 +1,2 @@
+Fix a :exc:`RuntimeWarning` when calling ``agen.aclose().throw(Exception)``
+Prevent ``agen.aclose()`` objects being re-used after ``.throw()``
```

Comment:
```suggestion
Fix a :exc:`RuntimeWarning` when calling ``agen.aclose().throw(Exception)``.
Prevent ``agen.aclose()`` objects being re-used after ``.throw()``.
```

Reply:
Ah I split this across two blurbs now so this is out of date